gaia-header.js

```diff
@@ -1,9 +1,10 @@
+/* jshint node:true */
+/* globals define */
```

Comment:
I think it worth to keep `.jslintrc` the same with other `gaia` repo, so there's no surprise between any sub projects